Mathlib/Algebra/MonoidAlgebra/Basic.lean

```diff
@@ -1203,6 +1203,10 @@ instance coeFun : CoeFun k[G] fun _ => G → k :=
 
 end AddMonoidAlgebra
 
+instance AddMonoidAlgebra.instIsCancelAdd [IsCancelAdd k] [AddZeroClass G] [IsCancelAdd G] :
+    IsCancelAdd (AddMonoidAlgebra k G) := by
+  delta AddMonoidAlgebra
+  exact Finsupp.instIsCancelAdd (α := G) (β := k)
```

Comment:
This shouldn't need to assume anything about `G`; I've corrected this (and the other instances) in #7582, though I'm alarmed the linter didn't spot this.

Reply:
I think this instance is now a duplicate

Reply:
deleted